client/app/cloud/project/billing/vouchers/addCredit/cloud-project-billing-vouchers-add-credit-agora.html

```diff
@@ -0,0 +1,28 @@
+<form novalidate
+      name="orderCreditForm"
+      data-ng-submit="$ctrl.order()">
+
+    <oui-modal data-title="{{:: 'cpb_vouchers_add_credit_title' | translate }}"
+               data-primary-label="{{:: 'cpb_vouchers_add_credit_valid' | translate }}"
+               data-secondary-action="$ctrl.cancel()"
+               data-secondary-label="{{:: 'common_cancel' | translate }}"
+               data-on-dismiss="$ctrl.cancel()"
+               data-loading="$ctrl.loading">
+
+        <oui-field data-label="{{:: 'cpb_vouchers_add_credit_quantity' | translate: { price: $ctrl.price.text } }}"
+                   data-ng-if="!$ctrl.loading">
+            <input class="oui-input"
+                   type="number"
+                   id="amount"
+                   name="amount"
+                   placeholder="{{:: 'cpb_vouchers_add_credit_amount' | translate }}"
+                   data-ng-model="$ctrl.amount"
+                   min="1"
+                   max="2000000000"
```

Comment:
Why not `200000000` ? 😄

Reply:
there is an upper limit in order funnel but it's not exposed by api :(

Reply:
Can be exported to a constant then ? 😃